roles/network_plugin/calico/defaults/main.yml

```diff
@@ -131,3 +131,6 @@ calico_allow_ip_forwarding: false
 
 # Calico IPAM strictaffinity
 calico_ipam_strictaffinity: false
+
+# Calico apiserver (only with kdd)
+calico_apiserver_enabled: false
```

Comment:
How about adding
```
- name: Check kdd calico_datastore if calico_apiserver_enabled
  assert:
    that: calico_datastore == "kdd"
  when:
    - calico_apiserver_enabled
```
to /network_plugin/calico/tasks/check.yml ?
Then we could make
```
calico_apiserver_enabled and calico_datastore == "kdd"
```
to simply
```
calico_apiserver_enabled
```
on most tasks later.

Reply:
Nice idea, it would make the code a bit cleaner indeed